docs/user-guide/walkthrough/k8s201.md

```diff
@@ -37,7 +37,7 @@ For example, here is the nginx Pod definition with labels ([pod-nginx-with-label
 Create the labeled Pod ([pod-nginx-with-label.yaml](/docs/user-guide/walkthrough/pod-nginx-with-label.yaml)):
 
 ```shell
-kubectl create -f docs/user-guide/walkthrough/pod-nginx-with-label.yaml
+kubectl create -f https://k8s.io/docs/user-guide/walkthrough/pod-nginx-with-label.yaml

```

Comment:
Please use a relative link rather than a hard one. Instead of:
`https://k8s.io/docs/user-guide/walkthrough/pod-nginx-with-label.yaml`
Use:
`docs/user-guide/walkthrough/pod-nginx-with-label.yaml`